default/python/AI/freeorion_tools/_freeorion_tools.py

```diff
@@ -170,3 +170,18 @@ def wrapper():
             this_turn = fo.currentTurn()
             return cache[this_turn] if this_turn in cache else cache.setdefault(this_turn, function())
     return wrapper
+
+
+def dict_to_tuple(dic):
+    return tuple(dic.iteritems())
+
+
+def tuple_to_dict(tup):
```

Comment:
This function have no documentation, has quite specific behavior, and used in single module.   I suggest to move it to context there it used.  In previous comments I suggest to remove it.